app/assets/stylesheets/main.sass

```diff
@@ -50,3 +51,31 @@ section.companies
 
     &:last-child
       margin-right: 0
+
+section.company
+  header
+    flex-direction: column
+
+  .detail
+    width: 100%
+    display: flex
+    flex-direction: row
+    flex-wrap: wrap
+    justify-content: space-between
+    align-items: baseline
+
+    .founded_at
+      flex: 1
+      text-align: left
+      min-width: 20rem
```

Comment:
Maybe not worth fixing but this rule should have been applied with a selector in `.detail`